### PR TITLE
fix: show container status instead of service status  in UI

### PR DESCRIPTION
**Pull request by tedim52**
## Description:
This change is the sister to this PR https://github.com/kurtosis-tech/kurtosis/pull/1560 which makes the same change in the CLI.

## Is this change user facing?
YES

## References: 
https://github.com/kurtosis-tech/kurtosis/pull/1560

### CODE REVIEW

File: engine/frontend/src/component/ServiceView.js [hunk 1]
```diff
@@ -65,7 +65,7 @@ const ServiceView = (service) => {
                 <Tbody>
                     {tableRow("Name", () => serviceInfo.name)}
                     {tableRow("UUID", () => <pre>{serviceInfo.serviceUuid}</pre>)}
-                    {tableRow("Status", () => statusBadge(serviceInfo.serviceStatus))}
+                    {tableRow("Status", () => statusBadge(serviceInfo.getContainer().Status()))}
```
Comment (tedim52) [inline engine/frontend/src/component/ServiceView.js]: Screen Shot 2023-10-16 at 3 15 18 PM
hmm for some reason Status isn't being returned in the service info ill check why

Comment (tedim52) [reply engine/frontend/src/component/ServiceView.js]: oops fixed

File: engine/server/webapp/asset-manifest.json [hunk 2]
```diff
@@ -1,13 +1,13 @@
 {
   "files": {
     "main.css": "./static/css/main.d56b8d0f.css",
```
Comment (h4ck3rk3y) [inline engine/server/webapp/asset-manifest.json]: wonder if all of this needs to be checked in

**Review by h4ck3rk3y — APPROVED** [review]

File: engine/server/webapp/asset-manifest.json (hunk 2 above)
Comment (tedim52) [reply engine/server/webapp/asset-manifest.json]: yes, these are files that are a result of rebuilding the frontend via `npm run build` in `engine/frontend`

**Pull request merged**